### PR TITLE
Add Send a Telegram Message GitHub Action

### PR DESCRIPTION
**Pull request by appleboy**
GitHub Link: https://github.com/appleboy/telegram-action
Marketplace: https://github.com/marketplace/actions/telegram-notify

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ Actions are triggered by GitHub platform events directly in a repo and run on-de
 
 #### Notifications and Messages
 
+ - [Send a Telegram Message](https://github.com/appleboy/telegram-action)
```
Comment (gep13) [inline README.md]: @appleboy thank you for taking the time to create this PR. As per the contributing guidelines, can I please ask that you don’t add your link at the top of the section, but rather at the bottom:

https://github.com/sdras/awesome-actions/blob/master/contributing.md#adding-to-this-list

Comment (appleboy) [reply README.md]: @gep13 Done.

Comment (gep13) [reply README.md]: @appleboy thank you for updating this!

**Comment by gep13** [comment]
@appleboy your changes have been merged, thanks for your contribution 👍

**Pull request merged**